### PR TITLE
Implement the filter by type (Network module)

### PR DESCRIPTION
**Pull request by clementprdhomme**
This PR implements the filter by type on the Network module.

## Acceptance criteria

- There is a way to filter content by its type: projects or organizations
  - The project filters only have effect on the projects
  - The organization filters only have effect on the organizations
  - Affects both the list and the map
- By default, none of the filters (and filter options) are selected

## Tracking

[ORC-239](https://vizzuality.atlassian.net/browse/ORC-239)

[ORC-239]: https://vizzuality.atlassian.net/browse/ORC-239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: client/src/app/(modules)/network/filters-sidebar.tsx [hunk 1]
```diff
@@ -10,11 +10,11 @@ import { Button } from '@/components/ui/button';
 import { Checkbox } from '@/components/ui/checkbox';
 import { Label } from '@/components/ui/label';
 
-import { useFilterSidebarOpen, useFilters } from './store';
+import { useNetworkFilterSidebarOpen, useNetworkFilters } from '../../../store/network';
```
Comment (Bluesmile82) [inline client/src/app/(modules)/network/filters-sidebar.tsx:13]: Is it possible to do a global import?

File: client/src/app/(modules)/network/page.tsx [hunk 2]
```diff
@@ -14,8 +14,9 @@ import { useSidebarScrollHelpers } from '@/containers/sidebar';
 import { Button } from '@/components/ui/button';
 import { Search } from '@/components/ui/search';
 
+import { useNetworkFilterSidebarOpen, useNetworkFilters } from '../../../store/network';
```
Comment (Bluesmile82) [inline client/src/app/(modules)/network/page.tsx:17]: Same here

File: client/src/app/(modules)/network/page.tsx [hunk 3]
```diff
@@ -57,21 +54,36 @@ export default function NetworkModule() {
     }
   }, [savedSidebarScroll, setSidebarScroll]);
 
+  // Focus back on the filters button when the filters sidebar is closed
```
[the remark this answers is not in the document]
Comment (clementprdhomme) [reply client/src/app/(modules)/network/page.tsx:59]: The button of the main sidebar is one of its first element so you have to tab all the way there to access it through the keyboard.

I'll fix the second issue though. You shouldn't be able to access the filters sidebar if it's collapsed.

File: client/src/app/(modules)/network/filters-sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      closeButtonRef.current.focus();
+    }
+  }, [filterSidebarOpen]);
+
+  if (!filterSidebarOpen) {
+    return null;
+  }
+
+  return (
+    <div className="absolute left-[calc(min(45%,_860px)_+_117px)] top-0 h-full w-[380px] border-l border-gray-200 bg-white p-12">
```
[the remark this answers is not in the document]
Comment (clementprdhomme) [reply client/src/app/(modules)/network/filters-sidebar.tsx]: This calculation is not in the code anymore 😉.

**Pull request merged**